build/configs/common_download.sh

```diff
@@ -31,6 +31,19 @@ CONFIG=${OS_PATH}/.config
 source ${CONFIG}
 SMARTFS_BIN_PATH=${BIN_PATH}/${CONFIG_ARCH_BOARD}_smartfs.bin
 
+for device in /dev/ttyACM*; do
+	if [ -c "$device" ]; then
+		DEFAULT_PORT=$device
+	fi
+done
+
+if [ -z ${DEFAULT_PORT} ]; then
+	for device in /dev/ttyUSB*; do
```

Comment:
ttyUSB0 will selected, but we normally use ttyUSB1 in rtl8730e.


Could you please make the defualt valuse of uart?

Reply:
No, it will select the highest port available
```
sonarqube@ws2030076986:~/Desktop/TizenRT$ sudo ./os/dbuild.sh download ALL
[sudo] password for sonarqube: 
Check Docker Image
Docker Image Version : tizenrt/tizenrt:2.0.0

 Port /dev/ttyUSB1 is selected

 ############################################
 WARNINGS:
 1. Make sure the board is in DOWNLOAD MODE.
 2. Make sure NO other application like putty,
 is occupying /dev/ttyUSB1.
 ############################################

```

Reply:
So if ttyUSB1 is highest among all ttyUSB*, it will select that

Reply:
Current change will sort the available port and assign the highest available port to defaulty_port